mlc_llm/utils.py

```diff
@@ -58,9 +58,16 @@ def argparse_add_common(args: argparse.ArgumentParser) -> None:
         "--quantization",
         type=str,
         choices=[*quantization_dict.keys()],
-        default=[list(quantization_dict.keys())[0]],
+        default=list(quantization_dict.keys())[0],
```

Comment:
FYI @MasterJH5574 this was throwing an error for me when using the default.

Reply:
Ah I just saw this. Is this still a problem now?

Reply:
No worries, I fixed it in the PR: https://github.com/mlc-ai/mlc-llm/pull/140/files#diff-1e0c77bf8bd4bb92e60a4746e2a26c492591b5704da300682510f60c50ca1dfaR54. Lmk if it looks good to you.